wp-admin/css/colors/_admin.scss

```diff
@@ -211,7 +196,14 @@ ul#adminmenu > li.current > a.current:after {
 	background: $menu-current-background;
 }
 
-#adminmenu li.wp-has-current-submenu div.wp-menu-image:before {
+#adminmenu li.wp-has-current-submenu div.wp-menu-image:before,
```

Comment:
Avoid using id selectors<br>Begin pseudo elements with double colons: `::`<br>Avoid qualifying class selectors with an element.<br>Selector should have depth of applicability no greater than 2, but was 4